src/components/Grid.jsx

```diff
@@ -33,6 +33,22 @@ class Grid extends React.Component {
     }
   }
 
+  getLines () {
+    let lines = []
+
+    for (let i = 0; i < this.props.hours; i++) {
```

Comment:
It’s all about single responsibility and composition. First, transform a given parameter (index) into a SVG line. Then generate a range of indexes and map over the previous function to aggregate multiple lines. In this case, we have range from 0 to x, so the range part can be simplified using `times` function as I demonstrated above.